includes/reader-activation/sync/class-esp-sync.php

```diff
@@ -110,6 +111,19 @@ public static function sync( $contact, $context = '' ) {
 
 		$contact = Sync\Metadata::normalize_contact_data( $contact );
 
+		// Go over all values in the contact's metadata and if they look like dates, then convert them from UTC to the site's timezone.
+		if ( ! empty( $contact['metadata'] ) ) {
```

Comment:
We're already filtering all contact metadata through [this `normalize_contact_data` method](https://github.com/Automattic/newspack-plugin/blob/trunk/includes/reader-activation/sync/class-metadata.php#L397) before we pass it to the ESP, which also applies a `newspack_esp_sync_normalize_contact` filter to the data to allow for greater control over the synced data, if needed. I think we should move the timezone handling to this method so that the filtered data includes this transformation, WDYT?